x-pack/test/functional_with_es_ssl/apps/cases/upgrade.ts

```diff
@@ -0,0 +1,288 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import {
+  getExternalServiceSimulatorPath,
+  ExternalServiceSimulator,
+} from '../../../alerting_api_integration/common/fixtures/plugins/actions_simulators/server/plugin';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default ({ getPageObject, getService }: FtrProviderContext) => {
+  const cases = getService('cases');
+  const kibanaServer = getService('kibanaServer');
+  const supertest = getService('supertest');
+  const testSubjects = getService('testSubjects');
+  const find = getService('find');
+
+  const updateConnector = async (id: string, req: Record<string, unknown>) => {
+    const { body: connector } = await supertest
+      .put(`/api/actions/connector/${id}`)
+      .set('kbn-xsrf', 'true')
+      .send(req)
+      .expect(200);
+
+    return connector;
+  };
+
+  const replaceNewLinesWithSpace = (str: string) => str.replace(/\n/g, ' ');
+
+  /**
+   * This test test an upgrade from 7.15 to the latest Kibana version.
+   * The x-pack/test/functional/fixtures/kbn_archiver/cases/7.17.5/case.json contains a case with
+   * all available user actions except user actions related to alerts. By importing the case,
+   * all migrations run. We ensure that the case shows all data after migrations.
+   */
+
+  describe('Upgrade', function () {
+    const CONNECTOR_ID = '92b970e0-09c4-11ed-9088-a95ba44d7455';
+    const CASE_ID = 'e26361b0-09c3-11ed-9088-a95ba44d7455';
+
+    before(async () => {
+      await kibanaServer.importExport.load(
+        'x-pack/test/functional/fixtures/kbn_archiver/cases/7.17.5/case.json'
+      );
+
+      await cases.testResources.installKibanaSampleData('logs');
```

Comment:
Is this to initialize the kibana instance? We don't depend on the logs for the tests right?

Reply:
The lens embeddable depends on them to be able to render. I am not sure how I can keep the lens consistent across versions.

Reply:
ah got it 👍